.gitignore

```diff
@@ -5,3 +5,6 @@
 *.img
 actual.out
 qemu.log
+tools/deptool/Cargo.lock
+tools/deptool/Cargo.lock
+tools/deptool/target
```

Comment:
Create a separate `.gitignore` file in tools?

Reply:
ok

Reply:
Done.